test/common_quantization.py

```diff
@@ -9,10 +9,12 @@
 
 import torch
 import torch.nn.quantized as nnq
+import torch.nn.quantized.dynamic as nnqd
 from common_utils import TestCase
 from torch.quantization import QuantWrapper, QuantStub, DeQuantStub, \
     default_qconfig, QConfig, default_observer, default_weight_observer, \
-    default_qat_qconfig
+    default_qat_qconfig, propagate_qconfig, convert, DEFAULT_DYNAMIC_MODULE_MAPPING
```

Comment:
We need to factor out all the mappings -- I will send a PR today. @jerryzh168

Reply:
Let me know if there is any update. Currently the mappings are still in this file.